modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -3480,6 +3480,63 @@ definition {
 		}
 	}
 
+	@description = "LPS-172918 - Verify that can trigger standalone action for a site scoped object"
+	@priority = 4
+	test CanTriggerStandaloneActionForSiteScopedObject {
+		task ("Given a site scoped object definition with a standalone action") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 172918",
+				objectName = "CustomObject172918",
+				pluralLabelName = "Custom Objects 172918",
+				scope = "site");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "true",
+				objectName = "CustomObject172918");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject172918");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 172918");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Action",
+				actionName = "ActionName",
+				active = "true",
+				errorMessage = "Error Message",
+				inputMethod = "true",
+				inputValue = "New Entry Test",
+				objectLabel = "Custom Object 172918",
+				thenAction = "Add an Object Entry",
+				whenAction = "Standalone");
+		}
+
+		task ("When the standalone action is triggered manually") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject172918");
+
+			ObjectAdmin.addObjectStringFieldEntryViaUI(entryValue = "Entry");
```

Comment:
I'm surprised that this works when we don't actually set the panel category key. Nothing to fix here, just an observation